portality/templates/admin/nav.html

```diff
@@ -1,13 +1,15 @@
 {% set journals = url_for("admin.index") %}
 {% set applications = url_for("admin.suggestions") %}
+{% set update_requests = url_for("admin.update_requests") %}
 {% set users = url_for("account.index") %}
 {% set site_search = url_for("admin.admin_site_search") %}
 {% set editor_groups = url_for("admin.editor_group_search") %}
 {% set bg = url_for("admin.background_jobs_search") %}
 
 {% set tabs = [
     (journals, "Journals"),
-    (applications, "Applications/update requests"),
+    (applications, "Applications"),
+    (update_requests, "Update Requests"),
```

Comment:
We should check if there are any downstream consequences to this.  Like, are there any pages that link to the applications search that will be tripped up by update requests now appearing under a new link?  The featuremap doesn't currently reach to all these corners of the system, so isn't itself likely to yield that answer, probably should just do some creative searching over the code to see.

Reply:
application_emails.py  def send_admin_ready_email(application, editor_id):